modules/button/react/lib/ToolbarDropdownButton.tsx

```diff
@@ -0,0 +1,89 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {colors, spacing, borderRadius} from '@workday/canvas-kit-react-core';
+import {focusRing, useTheme, Themeable, EmotionCanvasTheme} from '@workday/canvas-kit-react-common';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {ButtonColors} from './types';
+import {ButtonContainer} from './parts';
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+import {ToolbarIconButtonProps} from './ToolbarIconButton';
+
+export interface ToolbarDropdownButtonProps
+  extends Omit<ToolbarIconButtonProps, 'toggled' | 'onToggleChange'>,
+    Themeable {}
+
+const ToolbarDropdownButton = ({
+  theme = useTheme(),
+  buttonRef,
+  'aria-label': iconArialabel,
+  icon,
+  children,
+  ...elemProps
+}: ToolbarDropdownButtonProps) => {
+  const containerStyles = {
```

Comment:
is there a reason of keeping `containerStyles` inside a function?

Reply:
nope! I moved them up